src/main/java/com/iexec/worker/compute/app/AppComputeService.java

```diff
@@ -67,17 +71,22 @@ public AppComputeResponse runCompute(TaskDescription taskDescription,
                 workerConfigService.getTaskIexecOutDir(chainTaskId) + ":" + FileHelper.SLASH_IEXEC_OUT
         );
 
-        DockerRunResponse dockerResponse = dockerService.run(
-                DockerRunRequest.builder()
-                        .imageUri(taskDescription.getAppUri())
-                        .containerName(getTaskContainerName(chainTaskId))
-                        .cmd(taskDescription.getCmd())
-                        .env(env)
-                        .binds(binds)
-                        .maxExecutionTime(taskDescription.getMaxExecutionTime())
-                        .isSgx(taskDescription.isTeeTask())
-                        .shouldDisplayLogs(taskDescription.isDeveloperLoggerEnabled())
-                        .build());
+        DockerRunRequest runRequest = DockerRunRequest.builder()
+                .chainTaskId(chainTaskId)
+                .imageUri(taskDescription.getAppUri())
+                .containerName(getTaskContainerName(chainTaskId))
+                .cmd(taskDescription.getCmd())
+                .env(env)
+                .binds(binds)
+                .maxExecutionTime(taskDescription.getMaxExecutionTime())
+                .isSgx(taskDescription.isTeeTask())
+                .shouldDisplayLogs(taskDescription.isDeveloperLoggerEnabled())
+                .build();
+        // Enclave should be able to connect to the LAS
+        if (taskDescription.isTeeTask()) {
+            runRequest.setDockerNetwork(sconeLasConfiguration.getDockerNetworkName());
```

Comment:
Probably should have 
runRequest.addDockerNetwork(..)
instead, to allow multiple networks accessible from one container

Reply:
Good point yes!

Reply:
Actually, this is not possible, we can only use one network mode in the HostConfig. See [this](https://github.com/docker-java/docker-java/blob/master/docker-java-api/src/main/java/com/github/dockerjava/api/model/HostConfig.java#L176). Maybe there is another way.